### ISSUE TITLE
Bug on extracting Figma tokens

### ISSUE THREAD

**Issue by nicodeclercq** [diez/diez#45]
Hello, 
I have an issue on extracting my figma tokens, I have the following message in the console:
````
Figma authentication required.
ReferenceError: URLSearchParams is not defined
````
I have set the url in the .diezrc file and that's all (like you did in the video)
Is there something I am missing? How do you setup the auth?

thanks in advance

**Comment by roperzh** [comment]
Hey @nicodeclercq, thank you very much for reaching out! could you share with me:

- Your operating system
- Your node, yarn and diez versions?

I will try to dig into this in the meantime, but if you can share those it would be immensely helpful.

**Comment by roperzh** [comment]
Hey again! I was able to reproduce this issue in node versions < v10. I pushed #46 with a fix, and I will ping you again when a release with the fix is out.

**Comment by roperzh** [comment]
A fix for this was released in [v10.1.0](https://github.com/diez/diez/releases/tag/v10.1.0), feel free to reopen or open a new issue if you still have problems after upgrading.

**Comment by nicodeclercq** [comment]
Many thancks
